tls/s2n_config.c

```diff
@@ -68,13 +69,17 @@ static int s2n_config_setup_default(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default"));
     GUARD(s2n_config_set_signature_preferences(config, "default"));
+    GUARD(s2n_config_set_ecc_preferences(config, "default"));
+    GUARD(s2n_check_ecc_preferences_curves_list(config->ecc_preferences));
     return S2N_SUCCESS;
 }
 
 static int s2n_config_setup_tls13(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default_tls13"));
     GUARD(s2n_config_set_signature_preferences(config, "default_tls13"));
+    GUARD(s2n_config_set_ecc_preferences(config, "default_tls13")); 
+    GUARD(s2n_check_ecc_preferences_curves_list(config->ecc_preferences));      
```

Comment:
This is unnecessary at runtime. Our unit tests check all the preferences, and this check could get expensive if we add more curves / preference lists. If you want to verify outside of tests, do so from s2n_init.